frontend/src/hooks/api/organization/types.ts

```diff
@@ -9,3 +9,19 @@ export type RenameOrgDTO = {
   orgId: string;
   newOrgName: string;
 };
+
+export type CreateNewOrganization = {
```

Comment:
I think these types need `DTO` on them as well to be consistent with the naming convention used elsewhere if I remember correctly.